idaes/commands/env_info.py

```diff
@@ -27,7 +27,7 @@
     name="environment-info", help="Print information about idaes, OS, dependencies..."
 )
 @click.option("--solver", multiple=True, help="Add solvers to list of solvers to check")
-@click.option("--json", default=None, help="Write output ot a file")
+@click.option("--json", default=None, help="Write outputtoa file")
```

Comment:
This is wrong.

Reply:
I bear some responsibility for my constant inability to type "to" (mistyped calculates are also most likely me).